demo/demo_instructions.rst

```diff
@@ -25,10 +25,21 @@ There are three sites (Jacksonville, UCSD, Omaha) currently available on IARPA's
 
 Valid AWS credentials will need to be passed to the script to access the CORE3D dataset. To do this, use the ``--key_id`` and ``--secret_key`` arguments. It's possible that more sites will be added to the dataset in the future, so to view all available sites, use the ``--show-sites`` option. 
 
+Configuring Danesfield
+----------------------
+
+Two configuration files are needed for Danesfield. The first is required to be named ``input_<region>.ini``, where <region> is replaced by the name of the site. An example of what information it needs can be found at https://github.com/Kitware/Danesfield/blob/master/tools/example_vissat_config.json.
```

Comment:
Paragraph about input.ini needs a reference to https://github.com/Kitware/Danesfield/blob/master/input.ini, not to a JSON.